src/OpenTelemetry.Instrumentation.EntityFrameworkCore/CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Unreleased
 
+* Add new method parameter to support named options injection
```

Comment:
Please add a more descriptive entry. You could follow something similar to what you did for #1019

Reply:
Done